src/fermilib/utils/_unitary_cc.py

```diff
@@ -299,6 +392,9 @@ def uccsd_trotter_engine(compiler_backend=projectq.backends.Simulator()):
             circuit compiler, so that it may either simulate gates numerically
             or alternatively print a gate sequence, e.g. using
             projectq.backends.CommandPrinter()
+        qubit_graph(Graph): Graph object specifying connectivity of qubits.
+            The values of the nodes of this graph are qubits or weak qubit
+            references.  If None, all-to-all connectivity is assumed.
```

Comment:
Changed the use of the graph to only store the unique qubit IDs with WeakQubitRefs built on the fly in the filters to conform to this.